Style-Guide.md

```diff
@@ -43,6 +43,9 @@ Please don't use the following name for variable since it is used by our `entryp
 | planet-db-host 	| used in the docker script 	|
 | planet-db-port 	| used in the docker script 	|
 
+### il8n
+Planet uses the `il8n` translation module to translate our content and make it accessible. This module can be included in an html tag with the keyword il8n. For example, `<p il8n> Text to be translated </p>`. However, if there are no strings enclosed by the tag, or if only interpolation strings are enclosed, then the `il8n` attribute should not be used as it will raise errors during compilation.
```

Comment:
Let's remove the starting & ending spaces in the HTML tag example.  Those are mistakes that have unfortunately snuck into many of our HTML files.

Reply:
Ok sounds good!